controller.go

```diff
@@ -270,6 +285,21 @@ func (c *controller) NewNetwork(networkType, name string, options ...NetworkOpti
 
 	network.processOptions(options...)
 
+	if !c.started {
+		// Global stores are watched only when the controller enters the 'started' state.
```

Comment:
I'm not sure how to update this comment. @mavenugo, can you take a look?
> Also I don't see the equivalent reconciliation for endpoints

After restart docker daemon will only recreate networks, so I think reconciliation for endpoints is not needed?